client/components/mma/accountoverview/PersonalisedHeader.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
+import { headline, space } from '@guardian/source-foundations';
+import type { MembersDataApiResponse } from '../../../../shared/productResponse';
+import { isProduct, sortByJoinDate } from '../../../../shared/productResponse';
+
+interface PersonalisedHeaderProps {
+	mdapiResponse: MembersDataApiResponse;
+}
+
+export const PersonalisedHeader = ({
+	mdapiResponse,
+}: PersonalisedHeaderProps) => {
+	const userDetails = mdapiResponse.user;
+
+	const productDetails = mdapiResponse.products
+		.filter(isProduct)
+		.sort(sortByJoinDate);
+	const oldestProduct = productDetails[productDetails.length - 1];
+	const supportStartYear = new Date(oldestProduct.joinDate).getFullYear();
+
+	const calculatedTimeOfDay = () => {
+		const userDate = new Date().getHours();
```

Comment:
This might be clearer as `currentHour` or something similar as the variable isn't a complete date as the current name suggests.

Reply:
ah yeah good suggestion!